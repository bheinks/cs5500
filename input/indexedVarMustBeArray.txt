(* 
   Indexed variable must be of array type
*)

program p;

var x : integer;

begin
  read(x[4]);
  write('!')
end.

