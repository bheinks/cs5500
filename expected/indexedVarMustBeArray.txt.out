Line 10: Indexed variable must be of array type
